.gitlab/binary_build/otel_agent.yml

```diff
@@ -0,0 +1,16 @@
+---
+otel_agent_binary_build:
+  image: 486234852809.dkr.ecr.us-east-1.amazonaws.com/ci/datadog-agent-buildimages/deb_x64$DATADOG_AGENT_BUILDIMAGES_SUFFIX:$DATADOG_AGENT_BUILDIMAGES
+  tags: ["arch:amd64"]
```

Comment:
We would need to add `arm64` build. Internally staging cluster using ARM

Reply:
Sounds good, I'll add that 👍